implementations/index.md

```diff
@@ -75,7 +75,11 @@ has a page describing how to emit conformant JSON.
 
 ## Examples <a href="#examples" id="examples" class="headerlink"></a>
 
-* [RestPack::Serializer provides examples](http://restpack-serializer-sample.herokuapp.com/) which demonstrate sample responses.
+* https://www.balancedpayments.com/  was processing over a million dollars of credit card charges per day. Sadly they're shutting down. For now though, their documented test API calls still work, and their API docs show sample responses in a JSON-API stucture e.g. [Order objects](https://docs.balancedpayments.com/1.1/api/orders/#create-an-order)
```

Comment:
Unfortunately, balanced payments api is also following an old version of the spec (RC2), so it may not be worth featuring.